src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1074,9 +1075,8 @@ sub check_star_catalog {
     my $min_z = $z_ang_max;
 
     my $dither;			# Global dither for observation
-    if (defined $self->{DITHER_Y_AMP} and defined $self->{DITHER_Z_AMP}) {
-	$dither = ($self->{DITHER_Y_AMP} > $self->{DITHER_Z_AMP} ?
-		   $self->{DITHER_Y_AMP} : $self->{DITHER_Z_AMP}) * 3600.0;
+    if (defined $self->{cmd_dither_y_amp} and defined $self->{cmd_dither_z_amp}) {
+	$dither = max($self->{cmd_dither_y_amy}, $self->{cmd_dither_z_amp});
```

Comment:
Fortunately with hopper this kind of thing happens without even trying.  You always are checking against the state values that apply at that time.  :smile_cat:  But I'm not sure what you mean about *two* star catalogs.  Am I wrong in thinking that for acquisition there is only one acquisition (that gets done with "small" dither)?  The guide stars are acquired with small dither as well, and then dither gets changed.  So that would imply guide stars should be checked at some point (perhaps 8 minutes) into the observation when the "observation" dither is in place.